docs/Implementing-easyblocks.rst

```diff
@@ -254,6 +254,37 @@ redefining it entirely, since we call out to the original ``configure_step`` met
 Specific aspects of easyblocks
 ------------------------------
 
+.. _implementing_easyblocks_default_parameters:
+
+Default easyconfig parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+All of the parameters which are "set" in an easyconfig file (see :ref:`vsd_avail_easyconfig_params`)
+become key-value pairs in the ``self.cfg`` dictionary.  For instance, where the easyconfig file might
+specify
+
+.. code:: python
+
+  name = 'example'
+  version = '2.5.3'
+  versionsuffix = '-Python-3.7.4'
+
+you could access these within an easyblock via
+
+.. code:: python
+
+  ec_name = self.cfg['name']
+  ec_ver = self.cfg['version']
+  ec_suffix = self.cfg['versionsuffix']
+
+For these particular "universal" arguments, this mapping is done for you in the ``EasyBlock`` base
```

Comment:
I think what you mean here is that `self.name` is a valid class variable, right? As this stands it's a little confusing  because  if I have a custom variable `myvar`, I won't have `self.myvar` by default but I will always have self.cfg['myvar'] and I could then set that class variable in the constructor.

If that is what you mean, rather than use `ec_name` use `self.name` in the assignments to make this a little clearer.

Reply:
It _is_ confusing, not just as I wrote it here, but to a new developer trying to write an EasyBlock for the first time :-).   I've tried expanding the wording in this new section, and included a one-line usage example.

Reply:
That part looks good to me now